projects/plugins/boost/app/data-sync/Cornerstone_Pages_Entry.php

```diff
@@ -30,14 +30,15 @@ public function set( $value ) {
 
 		if ( empty( $value ) || count( $value ) === 1 && $value[0] === '' ) {
 			delete_option( $this->option_key );
+			$this->list_updated();
```

Comment:
I don't think this is necessary(this will trigger the generation twice). After the value is deleted, the next time this value is fetched, it will call `set` again with the new value. So, the regeneration will run because of line 41.
```suggestion
```

Reply:
I've removed the value deletion as it was causing unnecessary deletions when there was only a home page URL in the list.